Android/app/src/main/java/io/github/project_travel_mate/favourite/FavouriteCitiesFragment.java

```diff
@@ -19,6 +20,7 @@
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
+import butterknife.OnClick;
```

Comment:
do these need to be added here. I dont see any addition to this file which needs this